components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandler.java

```diff
@@ -87,6 +87,8 @@
     public void handle(HttpServletRequest request, HttpServletResponse response, AuthenticationContext context)
             throws FrameworkException {
 
+        // To keep track of whether particular request goes through the step based sequence handler.
+        context.setProperty(FrameworkConstants.STEP_BASED_SEQUENCE_HANDLER_TRIGGERED, true);
```

Comment:
If both DefaultStepBasedSequenceHandler and GraphBasedSequenceHandler set FrameworkConstants.STEP_BASED_SEQUENCE_HANDLER_TRIGGERED property to "true" what is the idea behind having this property ?